healthcheck/healthcheck.py

```diff
@@ -181,3 +182,26 @@ def _assess_overall_status(self):
             return False
 
         return True
+
+
+def _file_exists(path):
```

Comment:
@ekoval but the method is used by two different classes whose common parent is generic and doesn't necessarily care about files. Where would it go?
It's also just a generally useful utility function that would make sense in other contexts, it's not strictly tied to healthcheck.